src/rust/engine/fs/src/lib.rs

```diff
@@ -683,6 +711,7 @@ pub trait VFS<E: Send + Sync + 'static>: Clone + Send + Sync + 'static {
   }
 }
 
+#[derive(Debug, PartialEq)]
 pub struct FileContent {
   pub path: PathBuf,
```

Comment:
It's unclear to me whether this is expected to be an absolute path, a build_root relative path, or something else. @stuhood Can you clarify, and I'll add a comment (and maybe change my test)

Reply:
PathStat/Stat/etc should all be build_root relative.

Should probably rename the `build_root` member of PosixVFS to just `root`, since you'll be using it for other bases.